src/controllers/userProfileController.js

```diff
@@ -247,6 +246,8 @@ const userProfileController = function (UserProfile) {
       record.weeklySummariesCount = req.body.weeklySummariesCount;
       record.mediaUrl = req.body.mediaUrl;
       record.timeZone = req.body.timeZone;
+      record.hoursByCategory = req.body.hoursByCategory;
+      record.totalTangibleHrs = req.body.totalTangibleHrs;
```

Comment:
For this part you just moved record.hoursByCategory and record.totalTangibleHrs for clarity?

Reply:
It's because I cannot put hoursByCategory and totalTangibleHrs property in userProfile object unless you have perssion. For now, these two should be accessible for each role, so that I can tracking all the hours.